06_puppy/mohitnag/store_test.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type storer struct {
```

Comment:
I would call this a different name. It can make it a little confusing if you already have a `Storer` type, and then a `storer` type. 

Since this is encapsulating details about the test suite, you could call this `storerSuite`?

Reply:
done